lib/bowling.rb

```diff
@@ -0,0 +1,48 @@
+class Bowling
```

Comment:
Might want to look into splitting this class into several classes so that you stick to Separation of Concerns

Reply:
Not sure where else to add this comment so I will do it here. Might want to think about updating the README with your planning, how the game works, what the user must do to install everything to get it to work, etc, etc